Engine/Code/Engine/Input/Mouse.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "Engine/Core/CoreIncludes.h"
+
+class Mouse
+{
+	Vec2 m_PositionThisFrame;
```

Comment:
sure I get the purpose of this but what's the real purpose of this since it's not utilized anywhere right now?